config.example.yaml

```diff
@@ -31,8 +31,11 @@ fractalDeets:
 automod_regexes:
   - "^test(ing)?$"
   - "f[0o]+"
-logging_level: "WARN"
-logging_file: "latest.log"
+logging:
+  level: "WARN" # one of: INFO, WARN, ERROR, CRITICAL
```

Comment:
I may be wrong, but `logging.level` in the YAML is not being used in `logger.py`.

Reply:
Nevermind, I see the change in `patrick.py`. https://github.com/OpenRedstoneEngineers/Patrick/pull/12/files#diff-f5cab74a633a1f73d14057843158c64b6ea12aa87b1d0070a3bfdaf95a7133b4R317